src/utils/formatTime.js

```diff
@@ -1,13 +1,11 @@
-function formatDate (dateString, options) {
+// formats a date based on the options provided
+function formatDateTime (dateString, options) {
   const date = new Date(dateString)
-  options = { year: 'numeric', month: 'long', day: 'numeric' }
-  return date.toLocaleDateString('en-US', options)
-}
-
-function formatTime (timeString) {
-  const date = new Date(timeString)
-  const options = { hour: 'numeric', minute: 'numeric' }
-  return date.toLocaleTimeString('en-US', options)
+  if (options && (options.hour || options.minute)) {
```

Comment:
can you comment this to say why we need this conditional statement, or what the correct format of `options` is? might help a future programmer. alternatively, you can expose the specific options keys as function parameters.

Reply:
yes I can do that!

Reply:
there are so many options that I think it would be excessive to pass them as keys hah but I'm adding the comment now